### PR TITLE
Reject empty path

### LINKED ISSUE #2490: wrong files included
Hey,

For a project, I have the following configuration:

```php
// .php_cs.dist
<?php

declare(strict_types=1);

use PhpCsFixer\Config;
use PhpCsFixer\Finder;

$finder = Finder::create()
    ->in([
        __DIR__.'/src',
        __DIR__.'/tests'
    ])
;

return Config::create()
    ->setRiskyAllowed(true)
    ->setRules([...])
    ->setFinder($finder)
;
```

Yet, when I run it, I can see that the file `var/SymfonyRequirements.php` is affected. Is it the expected behaviour? (I wouldn't expect other directories than the one specified to be appended).

```
$ php -v
PHP 7.1.0-dev (cli) (built: Jan  3 2017 09:51:30) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.1.0-dev, Copyright (c) 1998-2016 Zend Technologies
```

PHP CS Fixer version you are using:
```
$ php-cs-fixer -V
PHP CS Fixer 2.0.0
```

### PR DESCRIPTION
**Pull request by julienfalque**
Close #2490.

### CODE REVIEW

File: tests/Console/ConfigurationResolverTest.php [hunk 1]
```diff
@@ -344,6 +344,29 @@ public function testResolvePathRelativeB()
...
+     */
+    public function testRejectInvalidPath(array $paths)
+    {
+        $resolver = $this->createConfigurationResolver(
+            ['path' => $paths],
+            null,
+            \dirname(__DIR__)
+        );
+
+        $this->expectException(InvalidConfigurationException::class);
```
Comment (kubawerlos) [inline tests/Console/ConfigurationResolverTest.php:358]: Can we test also for the message? That it starts with "Invalid path".

Comment (julienfalque) [reply tests/Console/ConfigurationResolverTest.php:358]: Done.

**Comment by SpacePossum** [comment]
Thank you @julienfalque.

File: src/Console/ConfigurationResolver.php [hunk 2]
```diff
@@ -380,6 +380,10 @@ public function getPath()
             } else {
                 $this->path = array_map(
                     static function ($path) use ($cwd, $filesystem) {
+                        if ('' === $path) {
```
Comment (theofidry) [inline src/Console/ConfigurationResolver.php:383]: is path trimmed beforehand?

Comment (julienfalque) [reply src/Console/ConfigurationResolver.php:383]: It's not, see #4954.

**Pull request merged**